File: Proyectos/Android/ExamenFragments_DavidCarroSalinas/app/src/main/java/com/example/examenfragments_davidcarrosalinas/Modelo/Libro.java
package com.example.examenfragments_davidcarrosalinas.Modelo;

import java.io.Serializable;

public class Libro implements Serializable {
    private int id;
    private String titulo;
    private String autor;
    private int numPags;

    public Libro(String titulo, String autor, int numPags) {
        this.titulo = titulo;
        this.autor = autor;
        this.numPags = numPags;
    }

    public Libro(int id, String titulo, String autor, int numPags) {
        this.id = id;
        this.titulo = titulo;
        this.autor = autor;
        this.numPags = numPags;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getAutor() {
        return autor;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public int getNumPags() {
        return numPags;
    }

    public void setNumPags(int numPags) {
        this.numPags = numPags;
    }

    @Override
    public String toString() {
        return "Libro{" +
                "id=" + id +
                ", titulo='" + titulo + '\'' +
                ", autor='" + autor + '\'' +
                ", numPags=" + numPags +
                '}';
    }
}

File: Desarrollo multimedia y dispositivos móviles/UT2/Teoría/recursosExamenAndroidDiciembre/kotlin/Libro.kt
package com.example.examenfragmentskotlin

import java.io.Serializable

class Libro(s: String, s1: String, i: Int) : Serializable {
    private var id = 0
    private lateinit var titulo: String
    private lateinit var autor: String
    private var numPags = 0

    fun Libro(titulo: String, autor: String, numPags: Int) {
        this.titulo = titulo
        this.autor = autor
        this.numPags = numPags
    }

    fun Libro(id: Int, titulo: String, autor: String, numPags: Int) {
        this.id = id
        this.titulo = titulo
        this.autor = autor
        this.numPags = numPags
    }

    fun getId(): Int {
        return id
    }

    fun setId(id: Int) {
        this.id = id
    }

    fun getTitulo(): String {
        return titulo
    }

    fun setTitulo(titulo: String) {
        this.titulo = titulo
    }

    fun getAutor(): String {
        return autor
    }

    fun setAutor(autor: String) {
        this.autor = autor
    }

    fun getNumPags(): Int {
        return numPags
    }

    fun setNumPags(numPags: Int) {
        this.numPags = numPags
    }

    override fun toString(): String {
        return "Libro{" +
                "id=" + id +
                ", titulo='" + titulo + '\'' +
                ", autor='" + autor + '\'' +
                ", numPags=" + numPags +
                '}'
    }
}